tests/test_absolute.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+import dpnp as inp
+
+import numpy
+
+
+@pytest.mark.parametrize("type",
+                         [numpy.float64],
+                         ids=['float64'])
+def test_absolute1(type):
+    n = 10**5
+    a = numpy.random.random(n)*2-1
+    ia = inp.array(a)
```

Comment:
Please do not use random in tests and use the same procedure for numpy and dpnp to generate input data.